runtime/bcverify/bcverify.c

```diff
@@ -2581,6 +2607,13 @@ j9bcv_verifyBytecodes (J9PortLibrary * portLib, J9Class * clazz, J9ROMClass * ro
 					ALWAYS_TRIGGER_J9HOOK_VM_METHOD_VERIFICATION_START(verifyData->javaVM->hookInterface, verifyData);
 				}
 
+#if defined(J9VM_OPT_VALHALLA_STRICT_FIELDS)
+				/* TODO Update the class version check if strict fields is released before value types. */
+				if (J9_IS_CLASSFILE_OR_ROMCLASS_VALUETYPE_VERSION(romClass) && isInitMethod) {
```

Comment:
You can add a new macro like `J9_CLASSFILE_VERSION_SUPPORTS_STRICT_FIELDS(romclass)` and define it to be `J9_IS_CLASSFILE_OR_ROMCLASS_VALUETYPE_VERSION(romclass)` for now. If strict field is release before value type, we can simply update the macro so that we don't need to update multiple places using `J9_IS_CLASSFILE_OR_ROMCLASS_VALUETYPE_VERSION`.